src/components/text-input-with-copy-button/TextInputWithCopyButton.scss

```diff
@@ -19,19 +19,25 @@
 
     .bdl-Button,
     .btn {
+        margin: 0;
+        border-left: 0;
+        border-radius: 0;
+        transition: background-color .5s ease, border-color .5s ease;
+    }
+
+    .bdl-Button:last-child,
+    .btn:last-child {
         min-width: 80px;
         margin: 0;
         border-left: 0;
-        border-top-left-radius: 0;
-        border-bottom-left-radius: 0;
+        border-radius: 0 6px 6px 0;
```

Comment:
@hkisel-box oops minor but we should use the SASS variables for this

not really blocking though but I can fix this later